resources/public/api/conf/3.0/common/errors.yaml

```diff
@@ -0,0 +1,236 @@
+      value:
+        code: RULE_PERIODIC_DATA_INCOMPLETE
+        message: One or more periodic updates are missing for this accounting period
+
+    ruleEndDateBeforeStartDate:
+      description: The accounting period end date value is before the start date.
+      value:
+        code: RULE_END_DATE_BEFORE_START_DATE
+        message: The supplied accounting period end date is before the start date
+
+    ruleAccountingPeriodNotSupported:
+      description: |
+        The specified accounting period is not supported, that is, the accounting period specified falls before the 
+        minimum tax year value.
+      value:
+        code: RULE_ACCOUNTING_PERIOD_NOT_SUPPORTED
+        message: |
+          The specified accounting period is not supported, that is, the accounting period specified falls before the
+          minimum tax year value
+
+    ruleDuplicateCountryCode:
+      description: The same countryCode cannot be supplied multiple times.
+      value:
+        code: RULE_DUPLICATE_COUNTRY_CODE
+        message: The same countryCode cannot be supplied multiple times
+
+    ruleBothPropertiesSupplied:
+      description: Both FHL and Non-FHL properties cannot be present at the same time.
+      value:
+        code: RULE_BOTH_PROPERTIES_SUPPLIED
+        message: Both FHL and Non-FHL properties cannot be present at the same time
+
+    ruleAccountingPeriodNotEnded:
+      description: The accounting period has not ended.
+      value:
+        code: RULE_ACCOUNTING_PERIOD_NOT_ENDED
+        message: The supplied accounting period has not ended
+
+    ruleAlreadyAdjusted:
+      description: Adjustments previously submitted against the supplied calculation ID.
+      value:
+        code: RULE_ALREADY_ADJUSTED
+        message: A summary may only be adjusted once. Request a new summary
+
+    ruleOverConsolidatedExpensesThreshold:
+      description: Adjustment to turnover exceeds consolidated expenses threshold.
+      value:
+        code: RULE_OVER_CONSOLIDATED_EXPENSES_THRESHOLD
+        message: The cumulative turnover amount exceeds the consolidated expenses threshold
+
+    ruleTradingIncomeAllowanceExceeded:
+      description: Adjustment to expenses where trading income allowance claimed.
+      value:
+        code: RULE_TRADING_INCOME_ALLOWANCE_CLAIMED
+        message: A claim for trading income allowance was made. Cannot also have expenses
+
+    rulePropertyIncomeAllowanceExceeded:
+      description: Adjustment to expenses where property income allowance claimed.
+      value:
+        code: RULE_PROPERTY_INCOME_ALLOWANCE_CLAIMED
+        message: A claim for property income allowance was made. Cannot also have expenses
+
+    ruleNoAccountingPeriod:
+      description: The supplied accounting period does not exist.
+      value:
+        code: RULE_NO_ACCOUNTING_PERIOD
+        message: The supplied accounting period does not exist
+
+    incorrectOrEmptyBodySubmitted:
```

Comment:
can we rename this to `ruleIncorrectOrEmptyBodySubmitted` (and update other yaml files accordingly) to be consistent with other `rule` errors